certified-connectors/Eagle/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2577 @@
+            },
+            "DESCR": {
+              "type": "string",
+              "description": "DESCR"
+            },
+            "NOTE": {
+              "type": "string",
+              "description": "NOTE"
+            },
+            "CUST_ID": {
+              "type": "string",
+              "description": "CUST_ID"
+            },
+            "STATUS": {
+              "type": "string",
+              "description": "STATUS"
+            },
+            "COMPLETED": {
+              "type": "string",
+              "description": "COMPLETED"
+            },
+            "PRIORITY": {
+              "type": "string",
+              "description": "PRIORITY"
+            },
+            "PIPE_PHASE": {
+              "type": "string",
+              "description": "PIPE_PHASE"
+            },
+            "APP_CONF": {
+              "type": "string",
+              "description": "APP_CONF"
+            },
+            "DUE_BY": {
+              "type": "string",
+              "description": "DUE_BY"
+            },
+            "DUE_TIME": {
+              "type": "string",
+              "description": "DUE_TIME"
+            },
+            "START_DATE": {
+              "type": "string",
+              "description": "START_DATE"
+            },
+            "START_TIME": {
+              "type": "string",
+              "description": "START_TIME"
+            },
+            "END_DATE": {
+              "type": "string",
+              "description": "END_DATE"
+            },
+            "END_TIME": {
+              "type": "string",
+              "description": "END_TIME"
+            },
+            "ACT_TIME": {
+              "type": "string",
+              "description": "ACT_TIME"
+            },
+            "BILL_TIME": {
+              "type": "string",
+              "description": "BILL_TIME"
+            },
+            "AUTHOR": {
+              "type": "string",
+              "description": "AUTHOR"
+            },
+            "HOURS": {
+              "type": "string",
+              "description": "HOURS"
+            },
+            "WORKCODE": {
+              "type": "string",
+              "description": "WORKCODE"
+            },
+            "MANAGER": {
+              "type": "string",
+              "description": "MANAGER"
+            },
+            "LOCATION": {
+              "type": "string",
+              "description": "LOCATION"
+            },
+            "ISNEW": {
+              "type": "string",
+              "description": "ISNEW"
+            },
+            "ISDELETED": {
+              "type": "string",
+              "description": "ISDELETED"
+            }
+          }
+        },
+        "headers": {
+          "Location": {
+            "description": "Location",
+            "type": "string"
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {},
```

Comment:
@doniyorniazov for triggers we can leave it like this if you want. We need properly defined responses for actions.